1228-public_page/user_navigates_to_landing_page_for_registration_and_can_see_files.feature

```diff
@@ -0,0 +1,51 @@
+Feature: User navigates to Landing Page for Registration and can see files
+
+    @1530
+    Scenario: Files that are Administrative Agreements are hidden
+        Given a User views the Landing Page for a Registration
+        And the Registration contains a File, which is an Administrative Agreement
+        When they view the Files section
+        Then they do not see the File that is an Administrative Agreement
+
+    Scenario: Files that are part of Registration are listed
+        Given a User views the Landing Page for a Registration
+        And the Registration contains Files
+        When they view the Files section
+        Then they can see files that are not Administrative Agreement are listed
+        And for each file they can see:
+            | name    |
+            | size    |
+            | version |
+            | license |
+        And they can download files that are not embargoed
```

Comment:
```suggestion
        Then they can see files that are not Administrative Agreements are listed
        And for each file they can see:
            | Name    |
            | Size    |
            | Version |
            | License |
        And they can download Files that are not Embargoed
```

Reply:
Er ikke `And they can download Files that are not Embargoed' neste scenario? Ser på det med testøyne og lurer på om jeg må skrive samme testen to ganger.

Reply:
hm jo, men man skal ikke kunne laste ned embargoed files, så det blir jo feil med bare `And they can download files` også(?)